PROFILER.md

```diff
@@ -9,21 +9,24 @@ which inspects the state of all threads to determine how long they have been
 running for. This does not require any code to be modified to support profiling
 so that any code which runs will be profiled automatically.
 
-The profiler can be enabled by setting the following:
+## Enabling Profiling
 
- * `com.iopipe.plugin.profiler` to `true`
- * `IOPIPE_PROFILER_ENABLE` to `true`
+### In General:
 
-The profiler may be customed by setting the specified properties and/or
+Set the environment variable `com.iopipe.plugin.profiler` to `true`
+### For Serverless Framework:
+
+In the `serverless.yml` file, under `functions:`, `(your_function_name_here):`, `environment:`, set the environwental variable `IOPIPE_PROFILER_ENABLE` to `true`
+
+The profiler may be customized by setting the specified properties and/or
```

Comment:
In this instruction, you mention properties, but you dropped it from bullets the docs below. One or the other (either mention properties and keep it in the docs, or drop the mention of properties)